services/graphql-server/src/graphql/definitions/platform/content/types/contact.js

```diff
@@ -75,6 +75,8 @@ input CreateContentContactPayloadMutationInput {
   lastName: String
   title: String
   status: Int = 2
+  primarySectionId: Int!
```

Comment:
Query for the section and use the siteId off of it. Throw an error if the site doesn't exist on the section.